docs/reference/commandline/daemon.md

```diff
@@ -768,6 +768,10 @@ the host, file ownership will have to be pre-arranged if the user or
 administrator wishes the containers to have expected access to the volume
 contents.
 
+To enable those advanced features for a specific container use ```-userns=host```
```

Comment:
If this option completely disables the userns option for a container, perhaps we should put it in its own section? Something like:

```Markdown

### Disable user namespace for a container

If you enable user namespaces on the daemon, all containers are started 
with user namespaces enabled. In some situations you might want to disable
this feature for a container, for example, to start a privileged container (see
[user namespace known restrictions](#user-namespace-known-restrictions)).

....
``` 

wdyt?